build.go

```diff
@@ -0,0 +1,111 @@
+package composer
+
+import (
+	"github.com/paketo-buildpacks/packit/v2"
+	"github.com/paketo-buildpacks/packit/v2/chronos"
+	"github.com/paketo-buildpacks/packit/v2/postal"
+	"github.com/paketo-buildpacks/packit/v2/scribe"
+	"os"
+	"path/filepath"
+	"time"
+)
+
+// Note that Go 1.18 requires faux 0.21.0 (https://github.com/ryanmoran/faux/releases/tag/v0.21.0)
+
+//go:generate faux --interface DependencyManager --output fakes/dependency_manager.go
+type DependencyManager interface {
+	Resolve(path, id, version, stack string) (postal.Dependency, error)
+	Deliver(dependency postal.Dependency, cnbPath, layerPath, platformPath string) error
+}
+
+type EntryResolver interface {
+	Resolve(name string, entries []packit.BuildpackPlanEntry, priorites []interface{}) (packit.BuildpackPlanEntry, []packit.BuildpackPlanEntry)
+	MergeLayerTypes(string, []packit.BuildpackPlanEntry) (launch, build bool)
+}
```

Comment:
Why have you opted not to use an entry resolver fake?

Reply:
It felt much simpler to just use the real Entry Resolver. I don't see a need to fake a stateless helper function with no side effects, and I felt like I was effectively stubbing out the behavior of the Entry Resolver. I tried to jumpstart a conversation here: https://paketobuildpacks.slack.com/archives/C02K4E2AUET/p1648215189613009, but there wasn't much engagement.

Reply:
@thitch97 thoughts? I see that Rob took a stab at investigating this here: https://github.com/paketo-buildpacks/cpython/pull/327

Reply:
I'm not sure that I'm convinced of the benefits of making this switch at the moment, but there is some conversation happening on the PR linked above. I'd prefer not to block this PR but I think the outcome of that conversation is what should inform this decision.